examples/notebooks/Creating Models/3-negative-particle-problem.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A step towards the Single Particle Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the [previous notebook](./2-a-pde-model.ipynb) we saw how to solve a PDE model in pybamm. Now it is time to solve a real-life battery problem! We consider the problem of spherical diffusion in the negative electrode particle within the single particle model. That is,\n",
+    "\\begin{equation*}\n",
+    "  \\frac{\\partial c}{\\partial t} = \\nabla \\cdot (D \\nabla c),\n",
+    "\\end{equation*}\n",
+    "with the following boundary and initial conditions:\n",
+    "\\begin{equation*}\n",
+    "  \\left.\\frac{\\partial c}{\\partial r}\\right\\vert_{r=0} = 0, \\quad \\left.\\frac{\\partial c}{\\partial r}\\right\\vert_{r=R} = -\\frac{j}{FD}, \\quad \\left.c\\right\\vert_{t=0} = c_0,\n",
+    "\\end{equation*}\n",
+    "where $c$ is the concentration, $r$ the radial coordinate, $t$ time, $R$ the particle radius, $D$ the diffusion coefficient, $j$ the interfacial current density, $F$ Faraday's constant, and $c_0$ the initial concentration. \n",
+    "\n",
+    "In this example we use the following parameters:\n",
+    "\n",
+    "| Symbol | Units              | Value                                          |\n",
+    "|:-------|:-------------------|:-----------------------------------------------|\n",
+    "| $R$      | m                | $10 \\times 10^{-6}$                            |\n",
+    "| $D$      | m${^2}$ s$^{-1}$ | $3.9 \\times 10^{-14}$                          |\n",
+    "| $j$      | A m$^{-2}$       | $1.4$                                          |\n",
+    "| $F$      | C mol$^{-1}$     | $96485$                                        |\n",
+    "| $c_0$    | mol m$^{-3}$     | $2.5 \\times 10^{4}$                            |\n",
+    "\n"
+   ]
+  },
```

Comment:
comment on the fact that all battery models in pybamm are dimensionless for better numerical conditioning and that this will be explained further in [link to notebook]